cachetory/serializers/msgpack.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from typing import Generic
+
+import ormsgpack  # type: ignore[import]
+
+from cachetory.interfaces.serializers import Serializer, ValueT
+
+
+class MsgPackSerializer(Serializer[ValueT, bytes], Generic[ValueT]):
+    """Uses the standard built-in `msgpack` serialization."""
```

Comment:
Technically, it isn't built-in, probably the copy-paste :)

Reply:
updated